base_export_manager/README.rst

```diff
@@ -2,12 +2,27 @@
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
 
-============================
-Manage model export profiles
-============================
+===========
+Base Export
```

Comment:
Maybe change back to `Manage model export profiles` now that we didn't rename module?